services/metadata_service/api/artifact.py

```diff
@@ -103,15 +105,17 @@ async def get_artifact(self, request):
             "405":
                 description: invalid HTTP Method
         """
-        flow_name = request.match_info.get("flow_id")
+        flow_id = request.match_info.get("flow_id")
```

Comment:
`flow_name` seems wrong based on current code (though I'm sure there is a history behind this).  So I fixed the var names in the code I touched.